Sources/BraintreeCore/BTAppContextSwitcher.swift

```diff
@@ -13,7 +13,10 @@ import UIKit
     /// The URL scheme to return to this app after switching to another app or opening a SFSafariViewController.
     /// This URL scheme must be registered as a URL Type in the app's info.plist, and it must start with the app's bundle ID.
     public var returnURLScheme: String = ""
-    
+
+    /// The URL to use for the PayPal universal link flow. Must contain the path `braintree-payments`.
```

Comment:
Nitpick - should we call this the PayPal app switch flow? Probably doesn't matter too much.

Also do we want to call out at the `BTAppContextSwitcher` level that setting the universalLink is only for PP and the urlScheme is only for Venmo?

Reply:
That's a great call - I don't think it can hurt to call it out. Updated here: https://github.com/braintree/braintree_ios/pull/1214/commits/22ab53a399841f4a0825c728527f4a3156a3c97f